CHANGELOG.md

```diff
@@ -26,13 +26,17 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
   - `all()` method moved from ArrayExtensions to SequenceExtensions. [#424](https://github.com/SwifterSwift/SwifterSwift/pull/424) by [n0an](https://github.com/n0an).
   - `none()` method moved from ArrayExtensions to SequenceExtensions. [#424](https://github.com/SwifterSwift/SwifterSwift/pull/424) by [n0an](https://github.com/n0an).
   - Added `any()` method to return if any element of sequence elements conforms to given condition. [#424](https://github.com/SwifterSwift/SwifterSwift/pull/424) by [n0an](https://github.com/n0an).
+- **SignedInteger**
+  - added `ordinalString(locale:)` method to return string ordinal representation of number in specified locale language. [#434](https://github.com/SwifterSwift/SwifterSwift/pull/434) by [n0an](https://github.com/n0an).
+- **SignedNumeric**
```

Comment:
I think we can put all the signed numeric under the same title :))

Reply:
But they're different. I've added one method to SignedNumeric and one to SignedInteger extension.
Should I specify both of them as SignedNumeric in CHANGELOG.md?